src/main.js

```diff
@@ -2,29 +2,27 @@
  * @Author: Brogan Miner <Brogan>
  * @Date:   2019-01-04T10:08:23-08:00
  * @Email:  brogan.miner@oregonstate.edu
- * @Last modified by:   Brogan
+ * @Last modified by:   Milan
```

Comment:
I believe there is an Atom (and possibly vscode) extension that modifies these headers automatically, with fresh dates and your name. Personally, I think we can slowly just delete these tags because git tracks this information on a commit-by-commit basis.